src/Components/PeriodicTable.css

```diff
@@ -0,0 +1,39 @@
+.periodic-table {
+    display: grid;
+    grid-template-columns: repeat(18, 90px);
+    grid-template-rows: repeat(10, 90px);
+    grid-gap: 4px;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+  }
+  
+  .element {
+    border: 1px solid black;
+    display: flex;
+    position: relative;
+    align-items: center;
+    justify-content: center;
+
+  }
+  
+  .element:hover {
+    transform: scale(1.25, 1.25);
+    z-index: 1;
```

Comment:
Do you really need this on hover? 🤔 
```suggestion

```

Reply:
Yes the hover highlights (at least on desktop) each Element, it just looks clean. And since I am doing the hover effect, the z-index is necessary. Otherwise the boxes of each element are overlapping while the hover effect is active.